pkg/u-boot/patches/patches-v2024.10-rc2/0002-0007-tweak-uefi-dt-selection.patch.patch

```diff
@@ -0,0 +1,25 @@
+From 41fe3e8f9f943bc53562a3f91f2cf48d3c7d0294 Mon Sep 17 00:00:00 2001
+From: =?UTF-8?q?Ren=C3=AA=20de=20Souza=20Pinto?= <rene@renesp.com.br>
+Date: Tue, 6 Aug 2024 15:03:02 +0200
+Subject: [PATCH 2/3] 0007-tweak-uefi-dt-selection.patch
```

Comment:
Fix the name of the patch: `0002-0007-tweak-uefi-dt-selection.patch.patch` -> `0002-tweak-uefi-dt-selection.patch.patch`